insolar-scripts/README.md

```diff
@@ -0,0 +1,30 @@
+# how to start launchnet and monitor
+
+To run launchnet, you must provide several files:
+
+`/scripts/insolard/bootstrap_template.yaml` - template for bootstrap config
```

Comment:
`/scripts` or ` ./insolar-scripts/` ?
the path starts with `/`. do you mean absolute path?

Reply:
no, its relative path, fix it